src/util/openfaas/index.js

```diff
@@ -37,6 +37,8 @@ const FAAS_READINESS_HTTP_FAILURE_THRESHOLD =
   process.env.FAAS_READINESS_HTTP_FAILURE_THRESHOLD || '5';
 const FAAS_READINESS_HTTP_SUCCESS_THRESHOLD =
   process.env.FAAS_READINESS_HTTP_SUCCESS_THRESHOLD || '1';
+const FAAS_PROFILE = process.env.FAAS_PROFILE || 'secure-function-profile';
```

Comment:
Can't we control it with a single variable like FAAS_PROFILE itself ?

Like:
```suggestion
const FAAS_PROFILE = process.env.FAAS_PROFILE || '';

if (FAAS_PROFILE !== "") {
	annotations["..."] = FAAS_PROFILE
}	
```